keyboards/lily58/keymaps/yshrsmz/rules.mk

```diff
@@ -0,0 +1,27 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = no        # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.
+SWAP_HANDS_ENABLE = no      # Enable one-hand typing
+
+# If you want to change the display of OLED, you need to change here
+SRC += ./lib/glcdfont.c
+SRC += ./lib/rgb_state_reader.c
+SRC += ./lib/layer_state_reader.c
+SRC += ./lib/logo_reader.c
+SRC += ./lib/keylogger.c
+#SRC += ./lib/mode_icon_reader.c
+#SRC += ./lib/host_led_state_reader.c
+#SRC += ./lib/timelogger.c
```

Comment:
This was fine as it was, what I meant is that all the stuff above except for `RGBLIGHT_ENABLE = yes` can be deleted.

Reply:
Sorry I completely misunderstood your intention.